tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-minversion = 3.8.6
+minversion = 3.9.0
```

Comment:
For other reviewers, `3.9.0` is the first release of `tox` that contains the https://github.com/tox-dev/tox/pull/1262 fix